### PR TITLE
apps/home: add microsite overview and detail pages and remove bg colo…

### PR DESCRIPTION
**Pull request by Rineee**
…r from overview page

No migrations added.

### CODE REVIEW

File: apps/home/models.py [hunk 1]
```diff
@@ -105,7 +105,9 @@ class HomePage(MetadataPageMixin, Page):
                      'apps_home.SimplePage',
                      'apps_gruenbuch.GruenbuchOverviewPage',
                      'apps_forms.ContactFormPage',
-                     'apps_forms.ParticipationFormPage']
+                     'apps_forms.ParticipationFormPage',
+                     'apps_home.MicrositeOverviewPage',
+                     'apps_home.MicrositeDetailPage']
```
Comment (phillimorland) [inline apps/home/models.py:110]: do we want the homepage to have microsite detail pages as direct children?

File: apps/home/templates/apps_home/overview_page.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 {% block content %}
 {% image page.intro_image max-1920x560 as intro_image %}
-<div class="fullwidth ds-block--fullwidth {% if page.background_color %}ds-block--{{ self.background_color }}{% else %}ds-block--body-bg{% endif %}">
+<div class="fullwidth ds-block--fullwidth ds-block--body-bg">
```
Comment (phillimorland) [inline apps/home/templates/apps_home/overview_page.html:6]: I think `ds-block--fullwidth ds-block--body-bg` can also go now as they were added to ensure colour was full width but might need testing

**Review by phillimorland — APPROVED** [review]
nice! just couple questions, if all find then good to have migrations added and be merged

File: apps/home/models.py (hunk 1 above)
Comment (Rineee) [reply apps/home/models.py:110]: I wasnt sure, but as we allow DetailPages as well and the editors will only be able to edit pages that are already there (as they are not allowed to edit homepage), I thought its more flexible this way?

File: apps/home/templates/apps_home/overview_page.html (hunk 2 above)
Comment (Rineee) [reply apps/home/templates/apps_home/overview_page.html:6]: Cool, will remove!

**Review by phillimorland — APPROVED** [review]
nice!

**Pull request merged**